### DISCUSSION TITLE
need details of using sta in yosys

### DISCUSSION THREAD

**Discussion by Vinayakamk** [YosysHQ/yosys#3897] [Q&A] [answered]
hello,
i want to do timing analysis in sta of yosys,,and wants to read sdc file in sta .

so need the little bit explaination how can i do that as i felt less info in manual. what are options and selection in this cmd
 **sta [options] [selection]**

i tried this.
yosys> sta
but wants to read sdc file 
expecting like yosys> sta -read_sdc my_constraints.sdc

**Comment by whitequark** [comment, marked as answer]
Yosys does not currently support SDC timing constraints for STA.
